server/src/main/java/org/eclipse/lsp/cobol/core/MessageServiceParser.java

```diff
@@ -20,13 +20,17 @@
 import org.antlr.v4.runtime.TokenStream;
 import org.eclipse.lsp.cobol.core.strategy.CobolErrorStrategy;
 
+import java.util.regex.Pattern;
+
 /**
  * Provide the support of message externalization for Parser.
  *
  * <p>Usage: options { superClass = MessageServiceParser;}
  */
 public abstract class MessageServiceParser extends Parser {
```

Comment:
I think you should add the DAF Trucks notice to the Contributors list in the copyright header